core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -69,9 +69,16 @@
 /* Normal memory, Outer Write-Back Write-Allocate Cacheable */
 #define TEE_MMU_TTB_RNG_WBWA    (1 << 3)
 
+/* Normal memory, Outer Write-Back no Write-Allocate Cacheable */
+#define TEE_MMU_TTB_RNG_WB      (3 << 3)
+
+#ifndef CFG_NO_SMP
```

Comment:
This fix is only relevant for that ARMv7 without LPAE, this issue does not seem to affect LPAE or ARMv8.